capa/features/extractors/dnfile_.py

```diff
@@ -83,13 +89,29 @@ def is_dotnet_file(self) -> bool:
         return bool(self.pe.net)
 
     def is_mixed_mode(self) -> bool:
+        assert self.pe is not None
+        assert self.pe.net is not None
+        assert self.pe.net.Flags is not None
+
         return not bool(self.pe.net.Flags.CLR_ILONLY)
 
     def get_runtime_version(self) -> Tuple[int, int]:
+        assert self.pe is not None
+        assert self.pe.net is not None
+        assert self.pe.net.struct is not None
```

Comment:
I'm a little concerned about code readability and maintainability of all these asserts (i.e. for field updates/changes)